<?xml version="1.0"?>
<launch>
  <!-- if show a image_view window subscribed to the generated stream -->
    <arg name="rosmastername" default="SATELLITE-S50-B"/>
    <machine name="facerecognition" address="facerecognition" user="root" password="ros_ros" env-loader="/catkin_ws/src/face_recognition_ros/env.sh $(arg rosmastername)" ssh-port="22"/>
  <node name="fr_node" machine="facerecognition" pkg="face_recognition_ros" type="find_faces.py" respawn="false" output="screen" required="true">
    <param name="image_input" value="camera/rgb/image_color"/>
    <param name="frameskipping" value="20"/>
    <param name="number_of_times_to_upsample" value="1"/>
    <param name="reorder_heads" value="True"/>
  </node>
</launch>
